src/base/bittorrent/session.cpp

```diff
@@ -3885,10 +3885,8 @@ void Session::handleTorrentFinished(TorrentHandle *const torrent)
                 emit recursiveTorrentDownloadPossible(torrent);
                 break;
             }
-            else {
```

Comment:
I would rather not include this type of changes in a "clean up" commit. Move it to another commit if you must.
Also I think doing this kind of changes just for the sake of cleaning up is dangerous. You should really revise the function as a whole than partially.

Reply:
But is the else not useless? With this change, if the `if` condition evaluates to true, we break out of the loop, if it is false the code below executes. If the code below is inside an `else` corresponding to the same `if`, the flow is the same.

Reply:
Please stop analyzing the code for a second and re-read my reply above. Also cleaning up like you have done here doesn't make much difference in terms of readability and (does a little harm in maintainability) IMO.

Reply:
Well I assumed such a simple control flow simplification could be considered a clean up. But no worries, I will undo this change.